docs/source/llms/llama-index/index.rst

```diff
@@ -221,6 +224,62 @@ You can disable tracing by running the same function with the ``disable`` parame
 FAQ
 ---
 
+
+How to log and load an index with external vector stores?
```

Comment:
Can we make it clearer that this will only work for indexes e.g. things that are compatible with at least one of the [engine types](https://github.com/mlflow/mlflow/blob/master/mlflow/llama_index/pyfunc_wrapper.py#L11). We still can't log LLMs/agents/etc.

Reply:
sure, added a note!